src/Microsoft.TemplateEngine.Core/EngineConfig.cs

```diff
@@ -32,5 +32,10 @@ public EngineConfig(IReadOnlyList<string> whitespaces, IReadOnlyList<string> lin
         public IReadOnlyList<string> Whitespaces { get; }
 
         public IDictionary<string, bool> Flags { get; }
+
+        public string OperationIdFlag(int operationId)
```

Comment:
Yep, I’ve removed all remnants of it.

From: Mike Lorbetske [mailto:notifications@github.com]
Sent: Tuesday, August 23, 2016 11:16 AM
To: seancpeters/microsoft-templateengine <microsoft-templateengine@noreply.github.com>
Cc: Sean Peters <sepeters@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [seancpeters/microsoft-templateengine] Refactor (#1)


In src/Microsoft.TemplateEngine.Core/EngineConfig.cs<https://github.com/seancpeters/microsoft-templateengine/pull/1#discussion_r75921731>:

> @@ -32,5 +32,10 @@ public EngineConfig(IReadOnlyList<string> whitespaces, IReadOnlyList<string> lin

>          public IReadOnlyList<string> Whitespaces { get; }

>

>          public IDictionary<string, bool> Flags { get; }

> +

> +        public string OperationIdFlag(int operationId)

I think this goes away too

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/seancpeters/microsoft-templateengine/pull/1/files/a425b2f9e55c8aa446463bd90060cb6bc99db3d0#r75921731>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT8a8bYD-9ogeiETZr29WPcivz3s8JWEks5qizjLgaJpZM4JqaeQ>.